dm/common/common.go

```diff
@@ -27,6 +27,9 @@ var (
 	// WorkerKeepAliveKeyAdapter used to encode and decode keepalive key.
 	// k/v: Encode(addr,name) -> time
 	WorkerKeepAliveKeyAdapter KeyAdapter = keyHexEncoderDecoder("/dm-worker/a/")
+	// WorkerConfigKeyAdapter the config path of upstream.
+	// k/v: Encode(source-id) -> addr
+	WorkerConfigKeyAdapter KeyAdapter = keyEncoderDecoder("/dm-worker/config/")
```

Comment:
Is `Encode(addr) -> source-id`? one usage is `clientv3.OpPut(common.WorkerConfigKeyAdapter.Encode(s.cfg.WorkerAddr), cfg.SourceID)`

Reply:
Yes. It was a mistake.